docs/governance/grants-bounties/bounties.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Bounties
+description:  >-
+  Explaination of bounties at Unlock Protocol
+---
+
+# Bounties
+
+There are a couple of different ways you can find work for bounties at
+Unlock Protocol. 
+
+The first way is to check out any of our github core protocol [repository](https://github.com/unlock-protocol/unlock/labels/%F0%9F%92%B0bounty). Head to the issues and filter for the label "bounty".
+
+Outside of the work we do inside github repositories we also have other types of 
```

Comment:
GitHub instead of github

QUESTION: are the GitHub bounties also represented in Wonderverse? if not, should they be?

Reply:
No and yes. I THINK they can be but haven't had a chance to play with the GitHub integration. It's on my radar.